src/color-template.ts

```diff
@@ -64,6 +64,16 @@ export const NightViewSettings: type.NormalColorSettings = {
     ],
 };
 
+export const NightGreenSettings: type.NormalColorSettings = {
+    type: 'normal',
+    backgroundColor: 'black',
+    foregroundColor: '#eeeeff',
+    strongColor: 'rgb(255,200,55)',
+    weakColor: '#aaaaaa',
+    radarColor: '#47a042',
+    contribColors: ['#efefef', '#d8e887', '#8cc569', '#47a042', '#1d6a23'],
```

Comment:
PR, Thanks.

I think it's better to reverse the light and dark in dark mode.

```suggestion
    contribColors: ['#444444', '#1B7D28', '#24A736', '#2DD143', '#57DA69'],
```

before:
![image](https://user-images.githubusercontent.com/15214393/148677798-0a3f20c0-abfb-4790-9e1e-c833cd30d89a.png)

after:
![image](https://user-images.githubusercontent.com/15214393/148677826-1ee472c5-ddc4-4acb-9ced-df03486a546f.png)

Reply:
Yeah, you are right. Looks better as you suggested :)